staging/src/k8s.io/apiserver/pkg/storage/storagebackend/config.go

```diff
@@ -99,6 +100,11 @@ type ConfigForResource struct {
 
 	// GroupResource is the relevant one
 	GroupResource schema.GroupResource
+
+	// The following fields hold config required/specific to KCP
+	//
+	// KcpIndicators hold some KCP indicators that are used by the reflector to instruct the storage layer how to extract the cluster name
+	KcpIndicators context.Context
```

Comment:
what is an indicator on a context? Having a ctx here looks strange. Convince me it isn't :-)

Reply:
essentially a struct that will hold some metadata for the storage layer - passed by the reflector.

Reply:
but why in a context? Why not typed?

Reply:
no specific reason, let's make it more concrete, I will change it to a struct.